src/backend/catalog/objectaddress.c

```diff
@@ -1496,89 +1498,6 @@ get_object_address_relobject(ObjectType objtype, List *object,
 	*relp = relation;
 	return address;
 }
-
```

Comment:
You raised the bar on best practices + quality by removing this code from the engine and replacing it with a hook. Great job!